exporter/datadogexporter/metrics/utils.go

```diff
@@ -62,9 +63,15 @@ func NewCount(name string, ts uint64, value float64, tags []string) datadog.Metr
 }
 
 // DefaultMetrics creates built-in metrics to report that an exporter is running
-func DefaultMetrics(exporterType string, hostname string, timestamp uint64) []datadog.Metric {
+func DefaultMetrics(exporterType string, hostname string, timestamp uint64, buildInfo component.BuildInfo, withCollectorVersion bool) []datadog.Metric {
```

Comment:
I think we should get rid of `withCollectorVersion` and just add it always

Reply:
agreed, updated